utilities/rali/rali_unittests/rali_unittests.cpp

```diff
@@ -43,8 +43,9 @@ using namespace cv;
 // #define TF_READER_DETECTION
 // #define CAFFE2_READER
 // #define CAFFE2_READER_DETECTION
-//  #define CAFFE_READER
+// #define CAFFE_READER
```

Comment:
Need to change all these to command line arguments instead of using #defines

Reply:
These changes will be added in a separate PR.